src/guardian/staking-module-guard/staking-module-guard.service.ts

```diff
@@ -156,6 +196,29 @@ export class StakingModuleGuardService {
     return attackIntersections;
   }
 
+  public async handleIntersectionBetweenUsedAndUnusedKeys(
```

Comment:
I would suggest to name the method something else, since it does not process this case in any way, but returns duplicates and outputs the log

Reply:
agree